src/ert/run_models/iterated_ensemble_smoother.py

```diff
@@ -51,35 +53,51 @@ def __init__(
             phase_count=2,
         )
         self.support_restart = False
-        self.ies_settings = analysis_config
+        self.analysis_config = analysis_config
         self.update_settings = update_settings
-        self._w_container = SIES(
-            len(simulation_arguments.active_realizations),
-            max_steplength=analysis_config.ies_max_steplength,
+
+        self.sies_step_length = functools.partial(
+            steplength_exponential,
             min_steplength=analysis_config.ies_min_steplength,
-            dec_steplength=analysis_config.ies_dec_steplength,
-        )
+            max_steplength=analysis_config.ies_max_steplength,
+            halflife=analysis_config.ies_dec_steplength,
+        )  # TO-DO: verify halflife exactly corresponds to dec in old implementation
```

Comment:
Can this TODO be an issue instead, or does it need to be completed before we can merge this PR?

Reply:
I think we should verify that `analysis_config.ies_dec_steplength` provides something meaningful to `halflife` before we merge the PR.